docs/config/README.md

```diff
@@ -22,6 +22,10 @@ The file should export an object containing options:
 
 ``` js
 // vue.config.js
+
+/**
+ * @type {import("@vue/cli-service").ProjectOptions}
```

Comment:
I recommend using single quotes here, to keep the style consistent with the rest of the document. And would you mind adding modifications to these two places(`ru/config/README.md`, `zh/config/README.md`) as well?

Reply:
Done...